### ISSUE TITLE
Configure statistics collection with command line properties

### ISSUE THREAD

**Issue by GoogleCodeExporter** [psi-probe/psi-probe#48]
```
History graphs like in Memory utilization has only 2 hours history time
frame. It would be nice to have larger history, for example 2 days. 

It could be simply parametrized by define on tomcat command line (for
example -Dhistory.timeframe=t) or better/later in probe web UI.

```

Original issue reported on code.google.com by `mschayna` on 6 Nov 2009 at 9:28
- Blocking: #106, #106
- Blocked on: #111
